src/gatsby/src/components/search/filter.js

```diff
@@ -58,10 +58,21 @@ const Filter = ({ filterItems, selectedFilterItems, handleFilterCheck, filterTyp
               checked={checked}
               onChange={event => {
                 handleFilterCheck(item, event)
+                if (typeof window.snowplow === 'function') {
+                  window.snowplow(
+                    "trackStructEvent",
+                    "Checkbox",
```

Comment:
The category could be something like 'park-search` or `find-a-park`.  It describes what part of the site you are measuring.